Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -587,6 +587,8 @@ extension PledgeViewController: PledgeShippingLocationViewControllerDelegate {
   ) {
     self.viewModel.inputs.shippingRuleSelected(shippingRule)
   }
+
+  func pledgeShippingLocationViewControllerLayoutDidUpdate(_: PledgeShippingLocationViewController) {}
```

Comment:
Is this part of a protocol? Do we need this function?

Reply:
Yeah - we conform to this in `PledgeViewController` and now in `RewardAddOnSelectionViewController`. The problem is that in the latter case the shipping location VC is contained in a table view header and so it needs to explicitly recalculate its size when it changes. In the case of `PledgeViewController` we don't need to do this but we're conforming to the same protocol 🤷‍♂️